luminoth/models/fasterrcnn/fasterrcnn_test.py

```diff
@@ -260,6 +277,247 @@ def testAnchors(self):
         self._assert_sequential_values(anchors[:, 2], config.anchors.stride)
         self._assert_sequential_values(anchors[:, 3], config.anchors.stride)
 
+    def testLoss(self):
+        """
+        Tests the loss of the FasterRCNN
+        """
+
+        # Create prediction_dict's structure
+        prediction_dict_random = {
+            'rpn_prediction': {},
+            'classification_prediction': {
+                'rcnn': {
+                    'cls_score': None,
+                    'bbox_offsets': None
+                },
+                'target': {},
+                '_debug': {
+                    'losses': {}
+                }
+            }
+        }
+        prediction_dict_perf = {
+            'rpn_prediction': {},
+            'classification_prediction': {
+                'rcnn': {
+                    'cls_score': None,
+                    'bbox_offsets': None
+                },
+                'target': {},
+                '_debug': {
+                    'losses': {}
+                }
+            }
+        }
+
+        # Set seeds for stable results
+        rand_seed = 13
+        target_seed = 43
+
+        # Set number of anchors
+        num_anchors = 1000
+
+        #
+        #   RPN
+        #
+
+        # Random generation of cls_score for rpn
+        rpn_cls_score = tf.random_uniform(
```

Comment:
Instead of generating random `cls_score` generate "opposite" scores to get high loss.